homework/index.js

```diff
@@ -0,0 +1,64 @@
+function getPromise() {
+  const ret = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
```

Comment:
Not really a reason to store this in a variable (especially if you just call it 'ret' or 'returnValue') - you can just return it right away.

Reply:
you are right. Variable not necessary